src/chttpd/src/chttpd.erl

```diff
@@ -1246,12 +1246,16 @@ start_span(Req) ->
         path_parts = PathParts
     } = Req,
     {OperationName, ExtraTags} = get_action(Req),
+    Path = case PathParts of
+        [] -> <<"">>;
+        [_ | _] -> filename:join(PathParts)
+    end,
     Tags = maps:merge(#{
         peer => Peer,
         'http.method' => Method,
         nonce => Nonce,
         'http.url' => MochiReq:get(raw_path),
-        path_parts => PathParts,
+        path_parts => Path,
```

Comment:
I'd prefer that this stayed a list but something between here and seeing it in Jaeger ended up concatenating it into a single string. I joined with `/`'s cause it was easy. Maybe serializing it with Jiffy would be better?

I'd like to keep this in some form since it shows what we're seeing after the decoding which is different than the `http.url` "raw" version.